cooperator_website_referral_source/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_referral_source_base_user,access_referral_source,model_referral_source,base.group_user,1,0,0,0
+access_referral_source_user,access_referral_source,model_referral_source,cooperator.cooperator_group_user,1,1,1,1
```

Comment:
Is that ok that `cooperator_group_user` have full right on the `referal.source` ?

Reply:
nope, I removed the unlink permission